hail/python/hail/vds/impex.py

```diff
@@ -0,0 +1,335 @@
+
+    new_header_file_path = hl.utils.new_temp_file('vds-append-to-header', 'txt')
+    with fs.open(new_header_file_path, 'w') as new_header_file:
+        new_header_file.write(extra_header)
+
+    vcf = to_merged_sparse_mt(VariantDataset(reference_data=ref, variant_data=var))
+    hl.export_vcf(vcf, output, append_to_header=new_header_file_path, parallel=parallel, metadata=metadata, tabix=tabix)
+
+
+@typecheck(
+    path=oneof(str, sequenceof(str)),
+    is_split=bool,
+    ref_block_fields=sequenceof(str),
+    infer_ref_block_fields=bool,
+    force=bool,
+    force_bgz=bool,
+    header_file=nullable(str),
+    min_partitions=nullable(int),
+    call_fields=oneof(str, sequenceof(str)),
+    reference_genome=nullable(reference_genome_type),
+    contig_recoding=nullable(dictof(str, str)),
+    array_elements_required=bool,
+    skip_invalid_loci=bool,
+    entry_float_type=enumeration(tfloat32, tfloat64),
+    filter=nullable(str),
+    find_replace=nullable(sized_tupleof(str, str)),
+    n_partitions=nullable(int),
+    block_size=nullable(int),
+    _create_row_uids=bool,
+    _create_col_uids=bool,
+)
+def import_vcf(
+    path,
+    *,
+    is_split=False,
+    ref_block_fields=(),
+    infer_ref_block_fields=True,
+    force=False,
+    force_bgz=False,
+    header_file=None,
+    min_partitions=None,
+    call_fields=('LPGT', 'PGT'),
+    reference_genome='default',
+    contig_recoding=None,
+    array_elements_required=False,
+    skip_invalid_loci=False,
+    entry_float_type=tfloat64,
+    filter=None,
+    find_replace=None,
+    n_partitions=None,
+    block_size=None,
+    _create_row_uids=False,
+    _create_col_uids=False,
+) -> VariantDataset:
+    """Import SVCR-VCF file(s) as a :class:`.VariantDataset`.
+
+    .. include:: _templates/experimental.rst
+
+    Notes
+    -----
+    The internal implementation of this method is a combination of :func:`hail.import_vcf` and
+    :func:`VariantDataset.from_merged_representation`. Refer to the documentation of those methods
+    for detailed usage of the parameters of both. The ``drop_samples`` parameter from
+    :func:`hail.import_vcf` is not present as a 'sites only' VDS is nonsense.
+
+    Note
+    ----
+    The following validations and transformations take place:
+
+    #. The ``LEN`` FORMAT field must exist and be of type :py:data:`.tint32`
+    #. One of ``GT`` or ``LGT`` must be a FORMAT field of type :py:data:`.tcall`
+    #. If ``is_split`` is ``False`` (the default), one of ``LA`` or ``LAA``, the local (alternate)
+       alleles field must be a FORMAT field of type ``array<int32>``.
+    #. ``LAA`` is transformed back into ``LA`` if ``LA`` is not already present, and then ``LAA`` is
+       dropped.
+    #. Entries are filtered to just those with present ``GT`` or ``LGT``.
+
+    Parameters
+    ----------
+    path : :class:`str` or :obj:`list` of :obj:`str`
+        One or more paths to VCF files to read. Each path may or may not include glob expressions
```

Comment:
Which is it lol?

Reply:
Good point, will fix both this and `import_vcf`